bin/dev

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env sh
 
-if ! command -v foreman &> /dev/null
-then
-  echo "Installing foreman..."
-  gem install foreman
-fi
```

Comment:
On second thought, it might be better and minimize conflicts to leave this file alone, and create a completely separate workflow for Docker/non-Heroku setups